components/in_memory_engine/src/write_batch.rs

```diff
@@ -470,14 +471,18 @@ impl WriteBatch for RegionCacheWriteBatch {
         Ok(())
     }
 
-    fn prepare_for_region(&mut self, region: CacheRegion) {
+    fn prepare_for_region(&mut self, region: metapb::Region) {
```

Comment:
So why not pass `region: &metapb::Region` instead, it should avoid some unnecessary clone call in the caller

Reply:
Fine